k8s/scripts/inspect.sh

```diff
@@ -47,6 +47,11 @@ function is_service_active {
   systemctl status "snap.$service" | grep -q "active (running)"
 }
 
+function collect_environment {
+  log_info "Copy environment file to the final report tarball"
+  cp -r --no-preserve=mode,ownership /etc/environment "$INSPECT_DUMP"
+}
```

Comment:
I would be careful collecting this. We know that API keys and other sensitive information can be stored in environment variables, and if the user uploads this report to Github, that's no good.

Can you grep for the specific lines we want and only export those instead?

Reply:
makes sense